tests/e2e/features/transactions/deposit/deposit-with-blockchain.feature

```diff
@@ -6,41 +6,56 @@ Feature: Deposit
 
   @id1418 @id1396
   Scenario: Make a deposit in ETH
-    Given I am on the Main page
-    Given I go to "Deposit" transaction section
-    When I click by "text" with "Your account" value
-    Then Element with "id" "amount-input" should have "" "value"
+    Given I go to page "/bridge?network=sepolia"
```

Comment:
Don't we setup env somewhere else in the configs? Why do we need to hardcode `?network=...`